### PR TITLE
Define ACL in terminology and link use to definition

### LINKED ISSUE #174: Add ACL to terminology section?
In [Resource Authorization](https://fcrepo.github.io/fcrepo-specification/#resource-authorization) we talk about ACLs alot -- I think it would be worth adding to terminology. Would mean we also have to reference WAC in the intro sentence.

Additionally, I think we should change the bibliography anchor SOLIDWEBAC to WAC, the spec is called simply [Web Access Control (WAC)](https://github.com/solid/web-access-control-spec#web-access-control-wac) even if its origins are SOLID.

### PR DESCRIPTION
**Pull request by zimeon**
Fixes #174

### CODE REVIEW

**Review by escowles — DISMISSED** [review]

**Comment by awoods** [comment]
The text of this PR looks good, however, I am seeing a Reference not found  in the references section.

File: index.html [hunk 1]
```diff
@@ -140,8 +140,15 @@
     <section id="terminology">
     <h2>Terminology</h2>
     <p>
-      Terminology is based on W3C's Linked Data Platform 1.0 [[LDP]] and Memento [[RFC7089]].
+      Terminology is based on W3C's Linked Data Platform 1.0 [[LDP]], Memento [[RFC7089]], and Web Access Control
+      [[SOLIDWAC]].
```
Comment (acoburn) [inline index.html]: My understanding is that the terminology section is normative, so I think the external references should be `[[!LDP]]`, `[[!RFC7089]]`, and `[[!SOLIDWEBAC]]`.

Also, it's `[[!SOLIDWEBAC]]` not `[[!SOLIDWAC]]`

File: index.html [hunk 2]
```diff
@@ -140,8 +140,15 @@
...
       <dl>
+        <dt><dfn>ACL</dfn>:</dt>
+        <dd>
+          An Access Control List Resource is a resource providing a set of authorization statements as defined in
+          [[SOLIDWAC]] <a href="https://github.com/solid/web-access-control-spec#access-control-list-resources">Access
```
Comment (acoburn) [inline index.html]: `[[SOLIDWAC]]` => `[[!SOLIDWEBAC]]`

Comment (zimeon) [inline index.html]: Thanks, will update PR

**Review by awoods — APPROVED** [review]

**Review by escowles — APPROVED** [review]

**Review by dannylamb — APPROVED** [review]

**Comment by awoods** [comment]
@zimeon : once the merge conflicts are resolved, I will merge this.

**Comment by awoods** [comment]
Superceded by: https://github.com/fcrepo/fcrepo-specification/pull/194

**Pull request closed without merging**